tests:testSwaggerCodegen failing

Test currently failing.  Appears to be breakage in upstream dependency.

$ ./gradlew :tests:testSwaggerCodegen
> Task :tests:testSwaggerCodegen FAILED

> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> A problem occurred configuring project ':swagger-java-client'.
> > Could not resolve all artifacts for configuration ':swagger-java-client:classpath'.
>    > Could not find any matches for com.android.tools.build:gradle:2.3.+ as no versions of com.android.tools.build:gradle are available.
>      Searched in the following locations:
>        - https://jcenter.bintray.com/com/android/tools/build/gradle/maven-metadata.xml
>        - https://jcenter.bintray.com/com/android/tools/build/gradle/
>      Required by:
>          project :swagger-java-client

Proposing to disable test case until we have time to find a real fix.

See https://openwhisk-team.slack.com/archives/C3TPCAQG1/p1544514441533000

--- a/tools/travis/runTests.sh
+++ b/tools/travis/runTests.sh
@@ -26,9 +26,7 @@ ROOTDIR="$SCRIPTDIR/../.."
 
 cd $ROOTDIR
 cat whisk.properties
-TERM=dumb ./gradlew :tests:testCoverageLean :tests:reportCoverage
-# disabled test until https://github.com/apache/incubator-openwhisk/issues/4169 is resolved
-#TERM=dumb ./gradlew :tests:testCoverageLean :tests:reportCoverage # :tests:testSwaggerCodegen
+TERM=dumb ./gradlew :tests:testCoverageLean :tests:reportCoverage :tests:testSwaggerCodegen
 
 bash <(curl -s https://codecov.io/bash)
 echo "Time taken for ${0##*/} is $SECONDS secs"

